artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/ActiveMQServerImpl.java

```diff
@@ -599,7 +599,7 @@ private void internalStart() throws Exception {
          return;
       }
 
-      configuration.parseSystemProperties();
+      configuration.parseProperties();
```

Comment:
the CLI would pass the properties file name into the ActiveMQServerIMPL.. and it would be passed here?